lib/utils/oclif-validation.ts

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2016-2020 Balena Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { ExpectedError } from '../errors';
+
+export function parseAsInteger(input: string, paramName?: string) {
```

Comment:
This one doesn't allow leading zeros ;-)

Reply:
Yeah, I saw that parseDbId one too.   Ok sure, can move to that validation.ts file.